packages/omuse-era5/setup.py

```diff
@@ -12,9 +12,8 @@
 url = 'https://github.com/omuse-geoscience/omuse'
 install_requires = [
     'omuse-framework',
-    'cdsapi'
```

Comment:
I don't think the omuse-era5 package will work without cdsapi installed. See https://github.com/esclapez/omuse/blob/updateInstall/src/omuse/community/era5/era5.py